build.gradle.kts

```diff
@@ -44,6 +44,12 @@ java {
     // IntelliJ run configuration.
     toolchain.vendor.set(JvmVendorSpec.ADOPTIUM)
 }
+// NEU's `error_prone_annotation` override gson to 2.2 which makes us lose some features.
```

Comment:
also what is error_prone_annotation lol, i cant find it in the neu project anywhere

Reply:
your guess is as good as mine, i'm not taking the 15 minutes to let the gradle cache rebuild the dependency map 🙏